### PR TITLE
REV-940: transaction declined message error

### PR DESCRIPTION
**Pull request by julianajlk**
[REV-940](https://openedx.atlassian.net/browse/REV-940) 
- Updated MFE to display the message for TransactionDeclined (as opposed to receiving it from Ecommerce)

### CODE REVIEW

File: src/payment/__snapshots__/PaymentPage.test.jsx.snap [hunk 1]
```diff
@@ -10780,7 +10780,7 @@ exports[`<PaymentPage /> Renders correctly in various states should render the b
                 </span>
                 <span>
                   <span>
-                    $0.00
+                    -$0.00
```
Comment (abutterworth) [inline src/payment/__snapshots__/PaymentPage.test.jsx.snap]: Everything looks good except these two snapshots where $0.00 becomes -$0.00. Seems unrelated, but worth looking into.

Comment (abutterworth) [reply src/payment/__snapshots__/PaymentPage.test.jsx.snap]: Oh just checked travis. It looks like you just need to update these snapshots (maybe there was a problem before that is now fixed.)

Comment (julianajlk) [reply src/payment/__snapshots__/PaymentPage.test.jsx.snap]: Added `<LocalizedPrice amount={discounts ? discounts * -1 : 0} />` to prevent passing -0 to LocalizedPrice

File: package.json [hunk 2]
```diff
@@ -115,7 +115,7 @@
     "react-test-renderer": "16.9.0",
     "reactifex": "1.1.1",
     "redux-mock-store": "1.5.3",
-    "rosie": "2.0.1",
+    "rosie": "^2.0.1",
```
Comment (robrap) [inline package.json]: @julianajlk: Why this change?  Thanks.
